src/store/ducks/lab/policy/sagas.ts

```diff
@@ -0,0 +1,91 @@
+import { call, put, select, takeLatest } from 'redux-saga/effects';
+import { AnyAction } from 'redux';
+
+import api from '../../../../services/api';
+import { AppState } from '../../..';
+import {
+  loadSuccess,
+  loadFailure,
+  createSuccess,
+  createFailure,
+  updateSuccess,
+  updateFailure,
+  deleteSuccess,
+  deleteFailure,
+} from './actions';
+import { LabPolicyTypes } from './types';
+
+const getToken = (state: AppState) => state.user.current.token;
+
+function* load(action: AnyAction) {
+  const { policyId, parameters } = action.payload;
+  const queryString = `/policies${policyId === 'all' ? '' : `/${policyId}`}`;
+  try {
+    const token: string = yield select(getToken);
+    const response: { [key: string]: any } = yield call(api.get, queryString, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+      params: parameters,
+    });
+    yield put(loadSuccess(policyId, response.data));
+  } catch (err) {
+    yield put(loadFailure());
+  }
+}
+
+function* create(action: AnyAction) {
+  const body = action.payload;
+  try {
+    const token: string = yield select(getToken);
+    yield call(api.post, '/policies', {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+      body,
+    });
```

Comment:
Currently, the headers aren't set correctly, and are instead included in the payload. Works when it's set to:
```suggestion
    yield call(api.post, '/policies', body, {
      headers: {
        Authorization: `Bearer ${token}`,
      },
    });
```

Reply:
Why does changing the order of headers and body help?

Reply:
![Screenshot from 2021-06-30 10-47-31](https://user-images.githubusercontent.com/39102698/123991972-cd6e2d80-d990-11eb-9aa5-c9734e658046.png)
As seen in the above screenshot, the entire object containing the Authorization header and the body is added into the payload, rather than the Authorization header being in the request headers. The third parameter of an axios call seems to be the payload, and the fourth parameter seems to be the options, including headers. So, fixing the parameters like shown fixes the issue. Please let me know if that makes sense. Thanks.

Reply:
I changed the approach of headers. Please check the latest commits.